docs/content/reports/ransomware.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Ransomware report"
+date: 2021-08-05T13:23:10+03:00
+draft: false
+description: "Provides information about ransomware simulation on your network"
+---
+
+{{% notice info %}}
+Check out [the documentation for other reports available in the Infection Monkey]({{< ref "/reports" >}}).
+{{% /notice %}}
+
+The Infection Monkey's **Ransomware Report** provides you with information and insight into ransomware simulation on your network.
+
+## Breach
```

Comment:
Let's add a section here giving a list of the sections, just like the Security Report page:
![image](https://user-images.githubusercontent.com/44770317/128541765-c86093e6-646a-45db-ba03-f632e6d3604a.png)